src/blacklist/5_0_0/sysd/blacklist_sysd.tex

```diff
@@ -127,10 +127,11 @@ \subsection{System functionalities and properties}
 \begin{itemize}
     \item Enables the application and other Core/Support systems to query the blacklist records that apply to them and are in force.
     \item Enables the application and other Core/Support systems to check if an arbitrary system is blacklisted.
+    \item Enables administrative Support systems to manage (query, create, remove) the blacklist entries.
 \end{itemize}
 
 \subsubsection {Non functional properties of the system}
--
+If an Authentication system is present in the Local Cloud, the Blacklist system will use its service(s) to verify a requester system before responding to any request. This verification is skipped when the requester is a system operator, or the request is a lookup for the identity service.
```

Comment:
"or the request is a lookup for the identity service". This part is unnecessary because it is related to the ServiceRegistry. Nobody will ask the blacklist server where to find the identity service.